src/porepy/fracs/fracture.py

```diff
@@ -190,7 +175,7 @@ def sort_points(self) -> np.ndarray:
         """Abstract method to sort the vertices as needed for geometric algorithms.
 
         Returns:
-            np.ndarray: Array of integeres containing the indices corresponding to the sorting.
+            Array of integers indices corresponding to the sorting.
```

Comment:
Maybe rephrase as "Array of integers, containing the indices corresponding to the sorting"?